react-component/index.js

```diff
@@ -35,18 +35,19 @@ export default {
       when: (data) => data.folder === 'custom',
```

Comment:
I think we should change this option data.folder === 'custom', to may be 

data.folder === 'custom-path',

Reply:
Done